.buildkite/bk.integration.pipeline.yml

```diff
@@ -53,16 +53,41 @@ steps:
           - fleet-privileged
           - upgrade
 
-  - group: "Stateful(Sudo): Ubuntu"
+      - label: "Win2022:non-sudo:{{matrix}}"
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'package-it'
+          .buildkite/scripts/integration-tests.ps1 {{matrix}} false
+        artifact_paths:
+          - build/**
+        agents:
+          provider: "gcp"
+          machineType: "n1-standard-8"
+          image: "family/platform-ingest-elastic-agent-windows-2022"
+        matrix:
+          - default
+
+  - group: "Stateful:Ubuntu"
     key: integration-tests-ubuntu
     depends_on:
-      - package-it
       - integration-ess
     steps:
-      - label: "x86_64:Group: {{matrix}}"
+      - label: "x86_64:non-sudo: {{matrix}}"
+        command: |
+          buildkite-agent artifact download build/distributions/** . --step 'package-it'
+          .buildkite/scripts/steps/integration_tests_tf_sudo.sh {{matrix}} false
```

Comment:
Seems like the scripts should be renamed since they test both sudo and non-sudo now

Reply:
renamed to `integration_tests_tf.sh`